pyproject.toml

```diff
@@ -2,3 +2,8 @@
 line-length = 88
 target-version = ['py39']
 include = '^.*\.py$'
+force-exclude = '''
```

Comment:
Why that? Just run "black ..." on all generated gql files after generation?

Reply:
currently the generated queries do not obey to line length settings. I.e., black will change the generated classes.

The longterm CI goal:

Run `make gql-introspection && make gql-query-classes` -> if `git diff` detects a change, then CI complains, because a schema change is not taken into account. Running black on generated classes complicates this more. Taking black out of the equation simplifies this a little: at least that was my initial thought.

However, I see your point in that we could also simply include black in the `make gql-query-classes` target and see it as a mandatory step of query class generation. That would be fine for CI too then :+1: Will adjust this.

Reply:
Just realizing that there is an issue with using black. Take this example:

```python
from pydantic import BaseModel, Extra, Field, Json # noqa: F401 # pylint: disable=W0611
```

becomes after black formatting

```python
from pydantic import (
    BaseModel,
    Extra,
    Field,
    Json,
)  # noqa: F401  # pylint: disable=W0611
```

The line breaks mess with the `noqa` annotation. Those annotations are only considered for the line they are placed on, i.e., black actually breaks the linting here. We could consider to adjust the generator to add those annotations on the file header, which would make the whole file disregard those linters. This would make linter checks less fine-grained though. Further, I feel that for now it would be easier to just avoid black on generated files.

Reply:
Actually I will adjust the generator to output already formatted headers. Should be fine then :+1:

Reply:
https://github.com/app-sre/qenerate/pull/20

Reply:
Ok generated classes are now also being modified by black. This works for now. If we stumble across another edge case, we can easily remove black formatting again from generated query classes.

Reply:
perfect, thank you